sechub-cli/build_go.sh

```diff
@@ -53,7 +53,7 @@ function init_go_modules() {
 }
 
 package_split=(${package//\// })
-package_name=${package_split[-1]}
+package_name=${package_split[*]: -1}
```

Comment:
With this, I found the answer in [this thread](https://stackoverflow.com/a/16109755/12464594)